saged/predict_tissue.py

```diff
@@ -323,6 +365,10 @@ def objective(trial, train_list, supervised_config,
                 if args.weighted_loss:
                     loss_weights = utils.calculate_loss_weights(train_data)
                     supervised_config['loss_weights'] = loss_weights
+                    if (supervised_config['name'] == 'BatchLogRegSKL' or
+                       supervised_config['name'] == 'LogRegSKL'):
```

Comment:
Do these exist anymore? It looks like you got rid of them.

Reply:
I guess not, the new versions used the pytorch and skl classes as their parents. Will remove this section